kythe/typescript/testdata/object.ts

```diff
@@ -21,13 +21,27 @@ const Object = {
 
   //- @method defines/binding Method
   //- Method.node/kind function
-  method() {}
+  method() {},
+
+  //- @"'sliteral'" defines/binding SLiteralProperty
+  //- SLiteralProperty.node/kind variable
+  'sliteral': 0,
```

Comment:
What if you have something like

```
  'sliteral#type': 0
```

I think our name generator needs to escape characters that aren't ascii or something.

Reply:
No, I think that's okay too. Kythe escapes such characters when processing it internally if that's what you're worried about, but for names it's fine. See the new test I pushed.